cmd/minishift/cmd/root.go

```diff
@@ -62,13 +64,26 @@ var viperWhiteList = []string{
 	"log_dir",
 }
 
+var (
+	homeDirExists bool
+	homeDirEmpty  bool
+)
+
 // RootCmd represents the base command when called without any subcommands
 var RootCmd = &cobra.Command{
 	Use:   "minishift",
 	Short: "Minishift is a tool for application development in local OpenShift clusters.",
 	Long:  `Minishift is a command-line tool that provisions and manages single-node OpenShift clusters optimized for development workflows.`,
 	PersistentPreRun: func(cmd *cobra.Command, args []string) {
 		var err error
+
+		if filehelper.Exists(constants.Minipath) {
+			homeDirExists = true
+		}
+		if filehelper.IsDirEmpty(constants.Minipath) {
```

Comment:
Might be a misunderstanding. I meant why Exist + IsEmptyDir? Hence my comment as well on the function as well. What is the outcome of IsEmptyDir is the path does not exist or is not a directory. I would assume it is _false_, unless you want to differentiate the case where the path does not exist with an error. Either way a call to IsEmptyDir should suffice here, right?